sunscreen_runtime/src/metadata.rs

```diff
@@ -198,7 +198,7 @@ pub struct FheProgramMetadata {
     pub name: String,
 }
 
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Clone, Serialize)]
```

Comment:
Can we not keep this `Deserialize`?

Reply:
I don't think Operation implements Deserialize